GroupIS1024/Pages/Index.cshtml

```diff
@@ -112,11 +114,16 @@
     <br />
     <!--Get in touch section-->
     <section class="get-in-touch">
-        <u><h2 style="margin-bottom: 30px;">Our office</h2></u>
-      
-            <div class="mapouter col-md-6 col-lg-6 col-sm-12 col-xs-12">
-                <div class="gmap_canvas" style="position:center: 20%;">
-                    <iframe width="500" height="400" id="gmap_canvas" src="https://maps.google.com/maps?q=university%20of%20cincinnati&t=&z=13&ie=UTF8&iwloc=&output=embed" frameborder="0" scrolling="no" marginheight="0" marginwidth="0"></iframe><a href="https://fmovies-online.net"></a><br><style>
+        <h2 style="margin-bottom: 30px;"><u>Our office</u></h2>
+
+        <div class="mapouter col-md-6 col-lg-6 col-sm-12 col-xs-12">
+            <div class="gmap_canvas" style="position:center;width:20%;">
+                <iframe width="500" height="400" id="gmap_canvas" src="https://maps.google.com/maps?q=university%20of%20cincinnati&t=&z=13&ie=UTF8&iwloc=&output=embed" frameborder="0" scrolling="no" marginheight="0" marginwidth="0"></iframe>
+                <a href="https://fmovies-online.net"></a>
+                <br />
+            </div>
```

Comment:
Closed the HTML tags properly as it is a good HTML practice.

Source: https://www.w3schools.com/html/html5_syntax.asp